### PR TITLE
Donaliu state

### PR DESCRIPTION
**Pull request by PakDLiu**
Added state field to send to coloAdapter
Matches machine states [here](https://msazure.visualstudio.com/One/_git/Compute-AzLinux-HanaRp?path=%2Fsrc%2Fcore%2Fcoloadapter%2Fcoloadapter.go&version=GBmaster&line=34&lineStyle=plain&lineEnd=34&lineStartColumn=1&lineEndColumn=39)

### CODE REVIEW

File: test/test_filter_hanarp_message.rb [hunk 1]
```diff
@@ -62,20 +63,21 @@ def test_filter_missing_machine_id
...
-                "stage" => "begin"
+                "event" => "Restart",
+                "stage" => "end"
             }
         ]
         filtered_records = filter(messages)
         data = JSON.parse(filtered_records[0]['message'])
         
-        assert_equal "Soft Shutdown", data['event']
+        assert_equal "Restart", data['event']
```
Comment (lagalbra) [inline test/test_filter_hanarp_message.rb:73]: Do you have a test for evaluating an Unknown case? If not, add one.

**Review by jovillas — APPROVED** [review]
LGTM

File: test/test_filter_hanarp_message.rb (hunk 1 above)
Comment (PakDLiu) [reply test/test_filter_hanarp_message.rb:73]: done

File: test/test_filter_hanarp_message.rb [hunk 2]
```diff
@@ -62,22 +63,49 @@ def test_filter_missing_machine_id
...
+        data = JSON.parse(filtered_records[0]['message'])
+        
+        assert_equal "Unknown Event", data['event']
+        assert_equal "Cisco_UCS:SJC2:testServiceProfile", data['data']['machineId']
+        assert_equal "1.1.1.1", data['data']['hostname']
+        assert_equal "4", data['data']['chassis']
+        assert_equal "7", data['data']['blade']
+        assert_equal "testServiceProfile", data['data']['serviceProfile']
+        assert_equal "begin", data['data']['stage']
+        assert_equal "unknown", data['data']['state']
```
Comment (lagalbra) [inline test/test_filter_hanarp_message.rb:106]: Shouldn't you assert that it's equal to the constant `Data.UNKNOWN`?

**Review by lagalbra — APPROVED** [review]

File: test/test_filter_hanarp_message.rb (hunk 2 above)
Comment (PakDLiu) [reply test/test_filter_hanarp_message.rb:106]: I can't see to access the Data class. I've tried Fluent::Plugin::HanarpMessage::Data::UNKNOWN and a bunch of other combinations.

**Pull request merged**